src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss

```diff
@@ -6,8 +6,7 @@
   position: absolute;
   background: $primary;
   top: 66px;
-  // NOTE: Currently, the sidebar takes 20% of the screen, thus 40% is half of map's occupation
-  left: calc((100% - #{$map-sidebar-width}) / 2);
+  left: 50%;
```

Comment:
@mike-lvov Can you please confirm that this is in line with your thoughts/expectations around the recent chat refactors.

Reply:
https://github.com/sparkletown/sparkle/pull/1202#discussion_r580994391